pyproject.toml

```diff
@@ -8,14 +8,14 @@ readme = "README.md"
 repository = "https://github.com/palaimon/ipyannotator"
 
 [tool.poetry.dependencies]
-python = "^3.8"
+python = ">=3.8,<3.11"
 pandas = "^1.1.2"
 traitlets = "^5.1.1"
 ipycanvas = "^0.10.2"
 ipyevents = "^0.8.0"
 ipykernel = "^5.3.4"
 ipywidgets = "^7.5.1"
-scikit-image = "^0.18.3"
+scikit-image = "^0.19.3"
 matplotlib = "^3.4.3"
 pydantic = "^1.8.2"
 PyPubSub = "^4.0.3"
```

Comment:
All of these requirements are over restrictive and contribute to the problems I mentioned in Issue #36. Unless you have specific reasons why you need to be restricting these libraries down to the patch level you are making it so that it is very difficult to use your software as anything but an application, and the documentation makes it seem like you want people to be using it along with their other code so that it can be useful in a workflow.

Reply:
@matthewfeickert 

Thanks for this great comment. I believe that `poetry add` caps requirements at patch level by default.
In any case, we don't cap our requirements for libraries by default when we set them manually. In the future we'll make sure to double check versioning when using tooling that creates/manages requirements.

@AlexJoz fyi

@itepifanio I agree with @matthewfeickert suggestion to switch from version cap to setting minimum version by default (also true for dev req.). I don't think we need to cap any of the ipyannotator requirements, but I'm not 100% sure.